package.json

```diff
@@ -18,7 +18,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
```

Comment:
@aditmehta9 and @annabauza , do we have to eject the react app to deploy to gh-pages? Is it possible to deploy without ejecting? I read [here](https://medium.com/curated-by-versett/dont-eject-your-create-react-app-b123c5247741) that ejecting create-react-app will make the app difficult to maintain in the future.

Reply:
This is test on pr and merge, deploy is in deploy.yml file